lib/cucumber/cli/configuration.js

```diff
@@ -18,8 +18,20 @@ function Configuration(options, args) {
     });
   }
 
-  var expandedFeaturePaths = Cucumber.Cli.FeaturePathExpander.expandPaths(unexpandedFeaturePaths);
-
+  var expandedFeaturePaths;
+  try {
+    expandedFeaturePaths = Cucumber.Cli.FeaturePathExpander.expandPaths(unexpandedFeaturePaths);
+  } catch (e) {
+      if (e.code === 'ENOENT' || e.code === 'ENOTDIR') {  
+        var errorMessage = 'No such file or directory - ';
+        errorMessage += path.relative(process.cwd(), e.path);
+        errorMessage += '. You can use `cucumber --init` to get started.';
+        console.error(errorMessage); 
+        process.exit(1);
```

Comment:
Again, I'd prefer to not exit from here directly. How about just `throw new Error(errorMessage)`? It results in the same thing but the parent could catch it if they wanted.